RecoLocalCalo/EcalDeadChannelRecoveryAlgos/interface/EcalDeadChannelRecoveryAlgos.h

```diff
@@ -7,16 +7,25 @@
 #include "DataFormats/EcalDetId/interface/EBDetId.h"
 #include <string>
 
-#include "RecoLocalCalo/EcalDeadChannelRecoveryAlgos/interface/EcalDeadChannelRecoveryNN.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
 
-template <typename DetIdT> class EcalDeadChannelRecoveryAlgos {
- public:
+#include "RecoLocalCalo/EcalDeadChannelRecoveryAlgos/interface/EcalDeadChannelRecoveryBDTG.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+template <typename DetIdT>
+class EcalDeadChannelRecoveryAlgos {
+public:
+  void setParameters(const edm::ParameterSet &ps);
   void setCaloTopology(const CaloTopology *topology);
-  EcalRecHit correct(const DetIdT id,
-                     const EcalRecHitCollection &hit_collection,
-                     std::string algo, double Sum8Cut, bool *AccFlag);
+  float correct(const DetIdT id,
+                const EcalRecHitCollection &hit_collection,
+                std::string algo,
+                double single8Cut,
+                double sum8Cut,
+                bool *accFlag);
 
- private:
-  EcalDeadChannelRecoveryNN<DetIdT> nn;
```

Comment:
This is just to be sure: backports into a production release must satisfy the "no change" rule. Here you remove the NN method for the deadchannel recovery, and substitute it with the BDTG one.
The default setting for ```recoverEBIsolatedChannels``` is ```false```, and I don't see in CMSSW any place or official workflow in which it is turned into ```true```. Therefore, I assume that the backport cannot modify the result in any of those official workflows.
Can you confirm that there are no use cases in 10_6_X that require such NN method for the ecal deadchannel recovery, and it id therefore safe to fully replace it with the BDTG one?